File: Assignment7/Program5.c

//Problem statements :write a program which accept number from user and display its table

#include<stdio.h>

void TableRev(int iNo)
{
    auto int icnt = 0;

    if(iNo < 0)
    {
        iNo = - iNo;
    }
   for(icnt =10; icnt >= 1; icnt--)
    {
       printf("%d ",icnt * iNo);
    }

}


int main()
{
    
    auto int iValue = 0;

    printf("Enter Number:\n");
    scanf("%d",&iValue);

    TableRev(iValue);   //function call

    return 0;
}

//Step :Test Cases 
/*
Input : 2
Output :20 18 16 14 12 10 8 6 4 2

Input :5
Output 50 45 40 35 30 25 20 15 10 5

Input :-5
Output 50 45 40 35 30 25 20 15 10 5


*/

File: Assignment32/Program5.java


import java.util.*;
//Write a java program which accept string from user and display it in reverse order
//Input :Marvellous 
//Output: "suollevraM"

 
class Program5
{
     
    //Function to calculate the Difference between small and capital Characters
    public static void Reverse(String str)
    {
        int icnt = 0;
        int iCount =0, iCount1 =0;;
        int Diff =0;
         char Arr[] = str.toCharArray();
         String Rev = " ";
        for(icnt = 0; icnt < Arr.length; icnt++)
        {
            
          Rev = Arr[icnt] + Rev;//adds each character in front of the existing string
          
        }
       System.out.println("Reversed "+Rev);
    }

   public static void main(String Args[]) //Entry Point Fucntion 
    {
        Scanner sobj = new Scanner(System.in);  //Scanner class object to Accept input
        String str = null;
        
        int iRet = 0;
        System.out.println("Enter String:"); 
        str = sobj.nextLine();
        
        
         Reverse(str);
         sobj.close();  

    }

}

